### PR TITLE
UIDATIMP-457: Fix Accessibility problems for settings/data-import/{-profiles} (Form elements must have labels)

### PR DESCRIPTION
**Pull request by IhorKoba**
## Purpose

To fix Accessibility problems related missing labels for settings/data-import/{...-profiles}

## Approach

- Add missing accessibility attributes to form elements 

## Refs

https://issues.folio.org/browse/UIDATIMP-457

### CODE REVIEW

File: src/components/MatchCriterion/edit/ExistingRecordSections/ExistingSectionFolio.js [hunk 1]
```diff
@@ -65,14 +67,23 @@ export const ExistingSectionFolio = ({
...
-            value={dropdownValue}
-            dataOptions={existingRecordFields}
-            onChange={handleExistingRecordSelect}
-            onFilter={handleFieldSearch}
-            dirty={isDirty}
-          />
+          <FormattedMessage
+            id="ui-data-import.match.existing.record.field"
+            values={{ recordType: existingRecordLabel }}
+          >
```
Comment (mariia-aloshyna) [inline src/components/MatchCriterion/edit/ExistingRecordSections/ExistingSectionFolio.js:73]: `existingRecordFieldLabel` prop is the same, consider converting this from `node` to `string` using `intl.formatMessage` method in the `MatchCriterion` component and use it here for `Section label` and `Selection aria-label`.

File: translations/ui-data-import/en.json [hunk 2]
```diff
@@ -93,6 +93,7 @@
   "match.qualifier.begins-with": "Begins with",
   "match.qualifier.ends-with": "Ends with",
   "match.qualifier.contains": "Contains",
+  "match.qualifier.text": "qualifier type text",
```
Comment (mariia-aloshyna) [inline translations/ui-data-import/en.json:96]:
```suggestion
  "match.qualifier.text": "Qualifier type text",
```

**Review by IvanKryzhanovskyi — APPROVED** [review]

File: src/components/MatchCriterion/edit/ExistingRecordSections/ExistingSectionFolio.js [hunk 3]
```diff
@@ -65,14 +67,23 @@ export const ExistingSectionFolio = ({
...
+                value={dropdownValue}
+                dataOptions={existingRecordFields}
+                onChange={handleExistingRecordSelect}
+                onFilter={handleFieldSearch}
+                dirty={isDirty}
+                aria-label={ariaLabel}
+              />
+            )}
+          </FormattedMessage>
+
```
Comment (mariia-aloshyna) [inline src/components/MatchCriterion/edit/ExistingRecordSections/ExistingSectionFolio.js:86]: Remove the blank line, please

File: src/components/MatchCriterion/edit/ExistingRecordSections/ExistingSectionFolio.js (hunk 1 above)
Comment (IhorKoba) [reply src/components/MatchCriterion/edit/ExistingRecordSections/ExistingSectionFolio.js]: done

File: translations/ui-data-import/en.json (hunk 2 above)
Comment (IhorKoba) [reply translations/ui-data-import/en.json]: done

File: src/components/MatchCriterion/edit/ExistingRecordSections/ExistingSectionFolio.js (hunk 3 above)
Comment (IhorKoba) [reply src/components/MatchCriterion/edit/ExistingRecordSections/ExistingSectionFolio.js]: done

**Review by mariia-aloshyna — APPROVED** [review]

**Pull request merged**